api/proto/agent/jobs.proto

```diff
@@ -0,0 +1,5 @@
+syntax = "proto3"
+
+    message Job {
```

Comment:
The controller and scheduler specs talk about actions, so you may want to use the same terminology here.